scripts/python/api/tnode.py

```diff
@@ -105,7 +105,7 @@ def cleanupDockerNetwork(self):
 
     # Remove all containers on this node
     def cleanupContainers(self):
-        self.runCmd("docker ps -a | grep alpine | awk '{print $1}' | xargs -r docker rm -fv ")
+        self.runCmd("docker ps -a | grep -v swarm | awk '{print $1}' | xargs -r docker rm -fv ")
```

Comment:
Should we explicitly kill the alpine and the skydns? We could have more infra containers other than swarm.